llama-index-integrations/llms/llama-index-llms-nvidia-ai-playground/llama_index/llms/nvidia_ai_playground/__init__.py

```diff
@@ -0,0 +1,3 @@
+from llama_index.llms.nvidia_ai_playground.base import NvidiaAIPlayground
+
+__all__ = ["NvidiaAIPlayground"]
```

Comment:
let's tentatively go with NVIDIA instead of NvidiaAIPlayground

Reply:
Changed.